cronet/build.gradle

```diff
@@ -1,94 +1,94 @@
-plugins {
```

Comment:
Revert this change. Use `-PskipAndroid=true` or set `skipAndroid=true` is your gradle.properties.